gnovm/tests/testdata/output_correct.txt

```diff
@@ -0,0 +1,20 @@
+# Test Output instruction correct
+
+exec $GNO test -verbose -root-dir=$ROOTDIR .
+
+! stdout .+
```

Comment:
Means we expect nothing from stdout.

Reply:
Can your useful GitHub comments be real file comments?

Reply:
sure ! 
f657673